src/main/java/ChatBot.java

```diff
@@ -0,0 +1,64 @@
+public class ChatBot {
+    private static final String INDENT = "    ";
+    private static final String NEW_LINE = INDENT + "____________________________________________________________ \n";
+    private static final String GREETING = INDENT + "Good day good sir! I am Chatimous Maximous, here to help you with your every need!\n";
+    private static final String BYE = "bye";
+    private static final String LIST = "list";
+    private static final String UNMARK = "unmark";
+    private static final String MARK = "mark";
+    private static final String EXIT = INDENT + "It has been a pleasure! I do hope to see you again! \n";
+    private static final String TODO = "todo";
+    private static final String DEADLINE = "deadline";
+    private static final String PRAISE = NEW_LINE + "A job well done Sir! \n" + NEW_LINE;
+    private static final int SEVEN = 7;
+    private static final String ENCOURAGEMENT = NEW_LINE + "Don't fret Sir! You'll get it soon \n" + NEW_LINE;
+    private static final int FIVE = 5;
+    private static final String EVENT = "event";
+    private static final String ADD_TASK = " has been accounted for! \n" + NEW_LINE;
+    private static final String NORMAL = "normal";
```

Comment:
I like how you used uppercase for all the constant names 👍

Reply:
Thank you for the help!